docs/packaging-an-application/template-functions.md

```diff
@@ -705,24 +687,6 @@ StatsdNetworkName() string
 
 StatsdNetworkName returns the name of the Statsd docker network.
 
-```yml
-services:
-  voting-app:
-    image: gaiadocker/example-voting-app-vote:good
-    environment:
-      REPLICATED_STATSD_ADDRESS: {{repl StatsdAddress }}
```

Comment:
Vendor will still need to add environment variables to access the Statsd container should we really remove this part as well?